src/main/resources/static/_shared.yml

```diff
@@ -3340,6 +3340,7 @@ components:
         - application
         - booking
         - assessment
+        - appeal
```

Comment:
would `assessmentAppeal` be clearer here? We've taken trouble elsewhere to  explicit that appeal relates to an assessment (see `APPROVED_PREMISES_ASSESSMENT_APPEALED` etc) as we expect that other things eg.  'bookings' are likely to be "appeal-able" in the future....